framework/tests/src/tests/ValueIgnoreRangeOverflowTest.java

```diff
@@ -0,0 +1,35 @@
+package tests;
+
+import java.io.File;
+import java.util.List;
+import org.checkerframework.common.value.ValueChecker;
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * Tests the constant value propagation type system without overflow.
+ *
+ * <p>NOTE: $CHECKERFRAMEWORK/framework/tests/value/ needs to be on the classpath. Otherwise
+ * ExceptionTest will fail because it cannot find the ExceptionTest.class file for reflective method
+ * resolution.
+ *
+ * @author plvines
```

Comment:
I don't know why this comment is here. I'm just going to delete it; I don't see any evidence that the claim it's making is true, either.